mgmtsystem_extended_hr/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Marcelo Frare (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+# Copyright 2019 Stefano Consolaro (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name":     "Management System - Extend HR",
```

Comment:
Module name should be `hr_mgmtsystem_nonconformity`, IMO

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#11modules

> When combining an Odoo module with another from OCA, Odoo's name goes before. I.e., if you want to combine Odoo's crm with OCA's partner_firstname, the name should be crm_partner_firstname.

Reply:
You know rules better than me, but are you sure that tis will be better? I'm extending NC with a field of HR, not HR with a field of NC.

Reply:
Yeah, I see your point.
Maybe `mgmtsystem_nonconformity_hr` instead?

We do need to remove the `extended` keyword, and clearify that we're extending `mgmtsystem_nonconformity`

Reply:
ok for mgmtsystem_nonconformity_hr